src/main/java/edu/wisc/my/restproxy/api/RestProxyConfiguration.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * 
+ */
+package edu.wisc.my.restproxy.api;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+/**
+ * {@link Configuration} activating necessary REST proxy services.
+ * 
+ * To use this class, simply {@link Import} it with the rest of your configuration.
+ * It's strongly suggested that your configuration provide a {@link org.springframework.web.client.RestTemplate} bean, but not required.
+ * 
+ * @author Nicholas Blair
+ */
+@Configuration
+@ComponentScan(value = {"edu.wisc.my.restproxy.dao", "edu.wisc.my.restproxy.service", "edu.wisc.my.restproxy.web" })
+public class RestProxyConfiguration {
```

Comment:
Yes and no. The instructions on `README.md` are still valid for those that use XML config. I'll add a section for those that have adopted `@Configuration` style instead.